.github/workflows/format.yml

```diff
@@ -5,10 +5,10 @@ on:
     branches:
       - main
     paths:
-    - 'mobile/**'
+      - 'mobile/**'
```

Comment:
could you fix the linting here - we should be checking this but currently arent - there is #24197 pending to fix